internal/cmd/operator_list_custom_resources.go

```diff
@@ -0,0 +1,76 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"k8s.io/apimachinery/pkg/util/duration"
+
+	"github.com/operator-framework/kubectl-operator/internal/cmd/internal/log"
+	"github.com/operator-framework/kubectl-operator/internal/pkg/action"
+)
+
+func newOperatorListCustomResourcesCmd(cfg *action.Configuration) *cobra.Command {
+	crLister := action.NewOperatorListCustomResources(cfg)
+	output := new(string)
+
+	cmd := &cobra.Command{
+		Use:     "list-custom-resources <operator>",
+		Aliases: []string{"list-crs"},
+		Short:   "List custom resources for an operator",
+		Long:    "List all custom resources on-cluster associated with a particular operator",
+		Args:    cobra.MaximumNArgs(1),
```

Comment:
We always expect exactly one argument for the operator name right?
```suggestion
		Args:    cobra.ExactArgs(1),
```

If you make this change, you don't have to check `len(args) == 0` in the `Run` function.